Plugin~/WebRTCPluginTest/GraphicsDeviceTest.cpp

```diff
@@ -24,22 +26,36 @@ TEST_P(GraphicsDeviceTest, CreateDefaultTextureV) {
     EXPECT_FALSE(tex->IsSize(0, 0));
 }
 
-#if defined(SUPPORT_SOFTWARE_ENCODER)
-TEST_P(GraphicsDeviceTest, CreateCPUReadTextureV) {
+TEST_P(GraphicsDeviceTest, CreateCPUReadTextureV)
+{
     const auto width = 256;
     const auto height = 256;
     const std::unique_ptr<ITexture2D> tex(device()->CreateCPUReadTextureV(width, height, m_textureFormat));
     EXPECT_TRUE(tex->IsSize(width, height));
     EXPECT_NE(nullptr, tex->GetNativeTexturePtrV());
     EXPECT_FALSE(tex->IsSize(0, 0));
 }
-#endif
 
-//[Note-sin: 2019-12-19] Real Unity Interface is required for testing the following functions, and it is not 
-//possible to create a dummy Unity interface (with its command buffer) on Metal devices
-#if !defined(SUPPORT_METAL) && !defined(SUPPORT_OPENGL_CORE)
+TEST_P(GraphicsDeviceTest, CreateTextureOnOtherThread)
```

Comment:
Create texture is called on current thread, Release texture is called on other thread.
so, I think it's a release timing test
```suggestion
TEST_P(GraphicsDeviceTest, ReleaseTextureOnOtherThread)
```